### PR TITLE
score-card bumped to 0.8.0

### PR DESCRIPTION
**Pull request by jvilimek**
## Hey, I just made a Pull Request!

Bump of [Score-card 0.8.0](https://www.npmjs.com/package/@oriflame/backstage-plugin-score-card/v/0.8.0) because of #21777

#### :heavy_check_mark: Checklist

- [x] ~A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))~ (n/a)
- [x] ~Added or updated documentation~ (n/a)
- [x] ~Tests for new functionality and regression tests for bug fixes~ (n/a)
- [x] ~Screenshots attached (for UI changes)~ (n/a)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by jvilimek** [comment]
Is this error related? Does not seem to: <https://github.com/backstage/backstage/actions/runs/7617519966/job/20746820135?pr=22454>

Can you please re-run?

**Comment by freben** [comment]
Nope that seems like a case of flakiness, retriggered

**Review by tudi2d — CHANGES_REQUESTED** [review]
Thanks for updating this - good to keep in sync! It would be nice to add a small changeset, which can be [similar to a `renovate` Changeset](https://github.com/backstage/backstage/pull/22452/files#diff-d6893e05da790ea8fb1807e7f2c7e62b98338fbfdc0dba11eacf5ee234fc7d02).

File: .changeset/dirty-mirrors-retire.md
Comment (tudi2d) [inline .changeset/dirty-mirrors-retire.md:4]: `example-app`* packages can be excluded as they are really just for debugging purposes in the main repository & not versioned/packaged - could have highlighted that in the review earlier, sorry.

```suggestion
```

**Review by tudi2d — CHANGES_REQUESTED** [review]
almost there - just a small fix to the changeset.

**Review by tudi2d — APPROVED** [review]

File: .changeset/dirty-mirrors-retire.md
Comment (tudi2d) [reply .changeset/dirty-mirrors-retire.md]: ok - just going to quickly apply the change to unblock this.

**Review by freben — APPROVED** [review]
Yepyep us maintainers have to do that.

Thanks for this!

**Pull request merged**
